lib/get-message.js

```diff
@@ -1,11 +1,30 @@
-const _ = require('lodash')
+const { defaultCommitMessages } = require('./default-commit-messages')
 
 module.exports = function (commitMessages, key, values) {
-  if (!Object.prototype.hasOwnProperty.call(commitMessages, key)) {
+  if (!Object.prototype.hasOwnProperty.call(commitMessages, key) &&
+  !Object.prototype.hasOwnProperty.call(defaultCommitMessages, key)) {
     throw new Error(`Unknown message key '${key}'`)
   }
 
   const templateValues = Object.assign({}, values)
 
-  return _.template(commitMessages[key])(templateValues)
+  function replaceMessageVariables (message, variables) {
+    let replacedString = message
+
+    Object.keys(templateValues).forEach((key) => {
+      replacedString = replacedString.replace(`\${${key}}`, variables[key])
+    })
+
+    return replacedString
+  }
+
+  let commitMessage = replaceMessageVariables(commitMessages[key], templateValues)
+
+  // if someone replaced the varibale name with something else,
+  // return the default message for that key
+  if (commitMessage.match(/\${.+?}/)) {
```

Comment:
Is this necessary? 🤔

Reply:
I think the switch from `_.template` to `String.replace` should make this addition not _necessary_, however from a user perspective the options are either:
* Not do this, and get raw `${foo}` text back with no reason why
* Do this, get the default message back with no reason why

Either one isn't ideal, but this way is probably better.

Reply:
Validating the greenkeeper config is a planned within the big Monorepo feature (https://github.com/greenkeeperio/greenkeeper/issues/594)
there we plan to at least sent an issue to the repo if there are errors in the config.